From 90ae13c7ab3df1f94df4cf30bd04b6d675606908 Mon Sep 17 00:00:00 2001
From: Bradley Zylstra <bzylstra@turnkeyscientific.com>
Date: Thu, 21 Aug 2025 13:30:12 -0400
Subject: [PATCH 5/6] added rustls-tls-webpki-roots bundled certs

---
 spotify_player/Cargo.toml | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/spotify_player/Cargo.toml b/spotify_player/Cargo.toml
index 840c888..3a8a39b 100644
--- a/spotify_player/Cargo.toml
+++ b/spotify_player/Cargo.toml
@@ -15,11 +15,11 @@ clap = { version = "4.5.41", features = ["derive", "string"] }
 config_parser2 = "0.1.6"
 crossterm = "0.29.0"
 dirs-next = "2.0.0"
-librespot-connect = { git = "https://github.com/librespot-org/librespot.git", branch = "dev", optional = true }
-librespot-core = { git = "https://github.com/librespot-org/librespot.git", branch = "dev" }
-librespot-oauth = { git = "https://github.com/librespot-org/librespot.git", branch = "dev" }
-librespot-playback = { git = "https://github.com/librespot-org/librespot.git", branch = "dev", optional = true }
-librespot-metadata = { git = "https://github.com/librespot-org/librespot.git", branch = "dev" }
+librespot-connect = { git = "https://github.com/librespot-org/librespot.git", branch = "dev", optional = true, default-features = false, features = ["rustls-tls-webpki-roots"] }
+librespot-core = { git = "https://github.com/librespot-org/librespot.git", branch = "dev", default-features = false, features = ["rustls-tls-webpki-roots"] }
+librespot-oauth = { git = "https://github.com/librespot-org/librespot.git",branch = "dev", default-features = false, features = ["rustls-tls-webpki-roots"] }
+librespot-playback = { git = "https://github.com/librespot-org/librespot.git", branch = "dev", optional = true, default-features = false, features = ["rustls-tls-webpki-roots"] }
+librespot-metadata = { git = "https://github.com/librespot-org/librespot.git", branch = "dev", default-features = false, features = ["rustls-tls-webpki-roots"] }
 log = "0.4.27"
 chrono = "0.4.41"
 reqwest = { version = "0.12.22", features = ["json"] }
-- 
2.50.1

